src/content/thought/the-state-of-ai-discourse.md

```diff
@@ -0,0 +1,85 @@
+
+Dunking on people or companies promoting AI may occasionally have merit, when there is clear exploitation or behaviour from bad actors, but it's important that we don't lose track of potentially relevant changes going on behind the scenes.
+
+There are plenty of skilled teams actively trying to build meaningful products that solve real world problems, but it's hard to focus solely on this when aspects of our industry are also being threatened. There is the hugely important debate of creativity in our industry, and how there is pride and enjoyment out of the act of creating vs shipping lines of code (which in case you're new here, is never a good metric of productivity or value).
+
+
+## Engaging productively
+
+You can of course continue to ignore AI wholesale, I wouldn't blame you for that. That approach may, however, leave you open to blind spots as the future of our ecosystem takes shape. Whether you like it or not, it has already infiltrated our toolbox, showing up in pretty much every software engineering tool there is to "help" you ship code. As always with tool diversification, there is a learning curve to figure out what is actually a useful augmentation of your creative process vs a complete blocker to getting things done.
+
+So how can we think differently about what's going on with AI?
+
+A good start is to **become more informed** about what's going on. Challenge nonsense when you see it. This doesn't have to (and probably shouldn't) be in public. Challenging views that appear to good to be true or present something as magical through your own research can only better your stance on something.
+
+**Celebrate those in the community trying to harness a sustainable and effective future for AI.** There are teams of people out there trying to make AI greener and more ethical than it is today, and those people need your support. Just like we're happy to support open source projects that enrich how we build things, it's important we support progressive AI work, too. Without those people, we'll be left with those who want to financially benefit from its existence as well as it's ability to exploit people and society as a whole.
+
+**Don't judge people who are not jumping on the hype.** There are a whole host of reasons why people may not be including AI in their process or product today. Maybe the value of adding such a tool is currently unclear, and there's work to be done to validate that before taking that step. It's always hard to remove functionality that isn't valuable or struggles to gain traction, so a considered approach is often sensible.
```

Comment:
Love this. Maybe mention that learning a whole new system that feels like it changes on the daily is too much for people who are super busy and/or uninterested